### ISSUE TITLE
Hound Necessary?

### ISSUE THREAD

**Issue by AlessandroMinali** [nesaulov/surrealist#42]
Hound is really annoying currently. Is it needed since rubocop is in Travis?

If so then we have two places now that we need to update any time with decide on new rules for linting.

**Comment by AlessandroMinali** [comment]
does Hound just use rubocop itself? I guess a fix is tehat it just uses the same config file we have at `.rubocop.yml`

**Comment by nesaulov** [comment]
I have introduced hound since rubocop went crazy when I added a separate gemfile for ruby 2.2.0 (see [this build](https://travis-ci.org/nesaulov/surrealist/jobs/294095392) for example)
> Running RuboCop...
Inspecting 4871 files

What I see in #34 is some kind of bug, it seems like hound didn't recognise `.rubocop.yml` and ran rubocop with community configuration.

I didn't manage to make rubocop work nicely through rake task, so I though HoundCI would be a better choice :( 

Maybe if you push code again, it will work as expected? That's the first time I face such behaviuor.
